projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -933,6 +934,7 @@ function ( &$value ) {
 					break;
 
 				case 'instant_search_enabled':
+				case 'jetpack_are_blogging_prompts_enabled':
```

Comment:
This looks wrong, it will overwrite the `instant_search_enabled` option!
```suggestion
```

Reply:
Ahh, thanks, now updated.